src/core/components/files-regular/cat-pull-stream.js

```diff
@@ -16,7 +16,8 @@ module.exports = function (self) {
     ipfsPath = normalizePath(ipfsPath)
     const pathComponents = ipfsPath.split('/')
     const restPath = normalizePath(pathComponents.slice(1).join('/'))
-    const filterFile = (file) => (restPath && file.path === restPath) || (file.path === ipfsPath)
+    const fileName = restPath.includes('/') ? restPath.split('/').pop() : restPath
+    const filterFile = (file) => (restPath && file.path === restPath) || (file.path === ipfsPath) || file.path === fileName
```

Comment:
ooh, nte to self I think this a bug in JS IPFS that needs ot be fixed in a separate PR

Reply:
https://github.com/ipfs/js-ipfs/pull/1920